### PR TITLE
Fix install

### PR DESCRIPTION
**Pull request by SimenB**
Without this change, it fails on install with:

```
> unleash-client@1.0.0-alpha.1 build /Users/simbekkh/repos/express-base/node_modules/unleash-client
> tsc src/*.ts -d --sourceMap --outdir ./lib --lib ES6,ES5

sh: tsc: command not found
```

If I install typescript, it still fails because the source code is not included in the package.

```
> unleash-client@1.0.0-alpha.1 build /Users/simbekkh/repos/express-base/node_modules/unleash-client
> tsc src/*.ts -d --sourceMap --outdir ./lib --lib ES6,ES5

error TS6053: File 'src/*.ts' not found.
```

Also allow this to be run on Node@7 without warnings (release imminent)

/cc @sveisvei @ivarconr

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "request": "^2.74.0"
   },
   "engines": {
-    "node": "6"
+    "node": ">=6"
```
Comment (SimenB) [inline package.json]: You test on node 4, though, why lock it to 6?

File: src/repository.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default class Repository extends EventEmitter implements EventEmitter {
     private storage: Storage;
     private etag: string;
 
-    constructor (backupPath: string, url: string, refreshIntervall?: number, StorageImpl = Storage) {
+    constructor (backupPath: string, url: string, refreshInterval?: number, StorageImpl = Storage) {
```
Comment (SimenB) [inline src/repository.ts]: Breaking change, I suppose...

File: examples/custom_strategy.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ class ActiveForUserWithEmailStrategy extends Strategy {
 
 const client = initialize({
     url: 'http://unleash.herokuapp.com/features',
-    refreshIntervall: 10000,
+    refreshInterval: 10000,
```
Comment (sveisvei) [inline examples/custom_strategy.js]: This client is not widely used as far as I know, so not worth it imo.

**Comment by sveisvei** [comment]
Add a console.log, and this is fine to merge and release another alpha.

**Comment by SimenB** [comment]
Just log, or map it over from the old name as well?

File: src/repository.ts (hunk 2 above)
Comment (SimenB) [inline src/repository.ts]: 👍

**Comment by sveisvei** [comment]
On second thought, this is a breaking change, lets throw instead as its startup time.

**Comment by SimenB** [comment]
Kk!

**Pull request merged**